linux_os/guide/services/ntp/service_timesyncd_root_distance_configured/bash/shared.sh

```diff
@@ -5,12 +5,14 @@
 # disruption = low
 
 config_file="/etc/systemd/timesyncd.d/oscap-remedy.conf"
-current_cfg_arr=( "/etc/systemd/timesyncd.conf" )
-current_cfg_arr+=("$(ls /etc/systemd/timesyncd.d/*)")
-# Comment existing NTP RootDistance settings
-for current_cfg in "${current_cfg_arr[@]}"
-do
-    sed -i 's/^RootDistanceMax/#&/g' "$current_cfg"
-done
+IFS=" " mapfile -t current_cfg_arr < <(ls -1 /etc/systemd/timesyncd.d/* 2>/dev/null)
+current_cfg_arr+=( "/etc/systemd/timesyncd.conf" )
+# Comment existing NTP FallbackNTP settings
```

Comment:
The old comment seem to be correct, the code below comments out `RootDistanceMax`, not `FallbackNTP`.

Reply:
Thanks for the note changed in 7286f5f